packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -101,11 +101,19 @@
 
 					<!-- Mapping section -->
 					<section class="form-section">
-						<h5 class="mb-1">
-							Calibration settings
-							<i v-tooltip="calibrationSettingsToolTip" class="pi pi-info-circle info-circle" />
-						</h5>
-						<p class="mb-2">Select one of the presets or customize the settings below.</p>
+						<h5 class="mb-1">Calibration settings</h5>
+						<div class="input-row">
+							<div class="label-and-input">
```

Comment:
Can those `<div class="label-and-input">` be removed by putting the input within the label?